src/m_files.cc

```diff
@@ -1025,13 +1035,30 @@ void Instance::M_WriteEurekaLump(Wad_file *wad) const
 	if (!loaded.portName.empty())
 		lump->Printf("port %s\n", loaded.portName.c_str());
 
+        //remember pwd
+        static char old_dir[FL_PATH_MAX];
+
+        if (getcwd(old_dir, sizeof(old_dir)) == NULL)
```

Comment:
`getcwd` is deprecated on Windows (there's the underscore equivalent available there). I wonder if we have a lib_file version here…

Reply:
As far as I could tell there isn't a lib_file version, but rolling one wouldn't be hard I don't think.